osu.Game/Beatmaps/MetadataUtils.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System.Linq;
+using System.Text;
+
+namespace osu.Game.Beatmaps
+{
+    /// <summary>
+    /// Groups utility methods used to handle beatmap metadata.
+    /// </summary>
+    public static class MetadataUtils
+    {
+        /// <summary>
+        /// Returns <see langword="true"/> if the character <paramref name="c"/> can be used in <see cref="BeatmapMetadata.Artist"/> and <see cref="BeatmapMetadata.Title"/> fields.
+        /// Characters not matched by this method can be placed in <see cref="BeatmapMetadata.ArtistUnicode"/> and <see cref="BeatmapMetadata.TitleUnicode"/>.
+        /// </summary>
+        public static bool IsRomanised(char c) => c <= 0xFF;
+
+        /// <summary>
+        /// Returns <see langword="true"/> if the string <paramref name="str"/> can be used in <see cref="BeatmapMetadata.Artist"/> and <see cref="BeatmapMetadata.Title"/> fields.
+        /// Strings not matched by this method can be placed in <see cref="BeatmapMetadata.ArtistUnicode"/> and <see cref="BeatmapMetadata.TitleUnicode"/>.
+        /// </summary>
+        public static bool IsRomanised(string? str) => str == null || str.All(IsRomanised);
```

Comment:
```suggestion
        public static bool IsRomanised(string? str) => string.IsNullOrEmpty(str) || str.All(IsRomanised);
```
Unless this was intended to not consider the empty case?

Reply:
`.All()` with a predicate actually returns `true` for empty enumerables, it's a logic/set-theory thing (see [sharplab](https://sharplab.io/#v2:C4LgTgrgdgPgAgJgAwFgBQiCM65IARyYB0AMgJZQCOA3OjpgJwAUARC0QIIA2XTAxgAsAhmCIBJAM4A5CAFsARgFMwAShXUgA===)). But I'll change as it's clearly a bit obtuse.